packages/opentelemetry-semantic-conventions-ai/opentelemetry/semconv_ai/__init__.py

```diff
@@ -236,3 +236,26 @@ class TraceloopSpanKindValues(Enum):
     AGENT = "agent"
     TOOL = "tool"
     UNKNOWN = "unknown"
+
+class MetricUtils:
+
+    def basic_metrics(meter: Meter):
```

Comment:
Can you also add return value types?

Reply:
Sound I would have to import Histogram and counter from metrics, will that be fine ?

Reply:
yes, sure

Reply:
Added, please verify